src/openrct2/localisation/StringIds.h

```diff
@@ -3911,6 +3911,7 @@ enum
 
     STR_COPY_INPUT_TO_CLIPBOARD = 6363,
 
+    STR_X_PERSON_DIED_ON_X = 6364,
     STR_NOTIFICATION_RIDE_CASUALTIES = 6464,
     STR_NOTIFICATION_RIDE_VEHICLE_STALLED = 6465,
```

Comment:
```suggestion
    STR_X_PERSON_DIED_ON_X = 6364,

    STR_NOTIFICATION_RIDE_CASUALTIES = 6464,
    STR_NOTIFICATION_RIDE_VEHICLE_STALLED = 6465,
```

We tend to leave a line break between different additions